dexios/src/subcommands/hashing.rs

```diff
@@ -7,7 +7,7 @@ use std::io::Read;
 // this hashes the input file
 // it reads it in blocks, updates the hasher, and finalises/displays the hash
 // it's used by hash-standalone mode
-pub fn hash_stream(files: &Vec<String>) -> Result<()> {
+pub fn hash_stream(files: &[String]) -> Result<()> {
```

Comment:
This is a good change - I don't know how I missed this

Reply:
thanks `clippy` for finding this :)